src/ResourceManager/Dns/Commands.Dns.Test/Commands.Dns.Test.csproj

```diff
@@ -63,7 +63,12 @@
     </Reference>
     <Reference Include="Microsoft.Azure.Management.Dns, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Dns.2.0.0-preview\lib\net452\Microsoft.Azure.Management.Dns.dll</HintPath>
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Dns.2.2.0-preview\lib\net452\Microsoft.Azure.Management.Dns.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="Microsoft.Azure.Management.Network, Version=17.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
@muwaqar if your test project is going to depend on the Network SDK, then you should update the Network entry in the [`TestMappings.json`](https://github.com/Azure/azure-powershell/blob/preview/TestMappings.json#L111-L117) file with a reference to the Dns test project so these tests get run whenever a change in Network is made